### PR TITLE
Switch cookies interaction to a WebDriver API

### LINKED ISSUE #645: Switch cookies interaction to a WebDriver API
**Describe the solution you'd like**
Switch for the supporting browser cookies interactions to a WebDriver

### PR DESCRIPTION
**Pull request by irynaVakulchyk**
Closes #645

### CODE REVIEW

File: vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Description: Integration tests for CookieManager functionality
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story:1]: `Description: Integration tests for cookie management in web application`

Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story:1]: Rename story: `HttpCookieManagerTest.story` -> `WebCookieSteps.story `

File: vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @epic vividus-plugin-rest-api
+
+Scenario: Verify step: "I set all cookies for current domain:$parameters"
+Given I am on the main application page
+When I set all cookies for current domain:
+|cookieName |cookieValue |path|
+|cookieName1|cookieValue1|/   |
+|cookieName2|cookieValue2|/   |
+Then a cookie with the name 'cookieName1' is set
+Then a cookie with the name 'cookieName2' is set
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story:13]: I believe it should be enough to set only one cookie

File: vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Scenario: Verify step: "I remove a cookie with the name '$cookieName' from the current domain"
+When I remove a cookie with the name 'cookieName1' from the current domain
+When I remove a cookie with the name 'cookieName2' from the current domain
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story:17]: This scenario depends on the previous one, please make them atomic

File: vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Description: Integration tests for CookieManager functionality
+
+Meta:
+    @epic vividus-plugin-rest-api
+
+Scenario: Verify step: "I set all cookies for current domain:$parameters"
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story:6]: `...Verify step: "When I...`

File: vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+Given I am on the main application page
+When I set all cookies for current domain:
+|cookieName |cookieValue |path|
+|cookieName1|cookieValue1|/   |
+|cookieName2|cookieValue2|/   |
+Then a cookie with the name 'cookieName1' is set
+Then a cookie with the name 'cookieName2' is set
+
+Scenario: Verify step: "I remove a cookie with the name '$cookieName' from the current domain"
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/HttpCookieManagerTest.story:15]: `...Verify step: "When I...`

File: vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/CookieManager.java [hunk 6]
```diff
@@ -65,43 +59,24 @@ public void addCookie(String cookieName, String cookieValue, String path, URI ur
...
     private void executeAddCookieScript(String cookieName, String cookieValue, String path, String domain)
     {
-        String cookie = String.format(JS_COOKIE_FORMAT, cookieName, cookieValue, path, domain);
-        LOGGER.debug("Adding cookie: {}", cookie);
-        javascriptActions.executeScript(cookie);
+        Cookie cookie = new Cookie.Builder(cookieName, cookieValue)
+                .domain("." + domain)
+                .path(path)
+                .build();
+        LOGGER.info("Adding cookie: {}", cookie.toString());
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/CookieManager.java:66]: - logger.atInfo
- just cookie

File: vividus-tests/src/main/resources/story/integration/WebCookieSteps.story [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+When I set all cookies for current domain:
+|cookieName |cookieValue |path|
+|cookieName1|cookieValue1|/   |
+Then a cookie with the name 'cookieName1' is set
+
+Scenario: Verify step: "When I remove a cookie with the name '$cookieName' from the current domain"
+Given I am on the main application page
+When I set all cookies for current domain:
+|cookieName |cookieValue |path|
+|cookieName |cookieValue |/   |
```
Comment (uarlouski) [inline vividus-tests/src/main/resources/story/integration/WebCookieSteps.story:17]: please use values different than parameter names

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/CookieSteps.java [hunk 8]
```diff
@@ -194,7 +194,6 @@ public void setAllCookies(ExamplesTable parameters)
         {
             cookieManager.addCookie(row.get("cookieName"), row.get("cookieValue"), row.get("path"), currentUrl);
         });
-        navigateActions.refresh();
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/CookieSteps.java:197]: 1. Why it is nor required?
2. If it still not, step description should be updated

**Pull request merged**